packages/dd-trace/src/dcitm.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const dc = require('diagnostics_channel')
+
+const CHANNEL_PREFIX = 'dd-trace:bundledModuleLoadStart'
+
+if (!dc.subscribe) {
+  dc.subscribe = (channel, cb) => {
+    dc.channel(channel).subscribe(cb)
+  }
+}
+if (!dc.unsubscribe) {
+  dc.unsubscribe = (channel, cb) => {
+    dc.channel(channel).unsubscribe(cb)
```

Comment:
unsubscribe method is treacherous :S
would be safer to check if it has subscribers?
```
if (dc.channel(channel).hasSubscribers) dc.channel(channel).unsubscribe(cb)
```

Reply:
Yes I think that would also make the intent clearer.

Reply:
Are there scary side effects? I'm not averse to checking for subscribers, was mostly trying to emulate existing behavior here.

Reply:
The function may not always be present on older versions of Node.js. It existed on the ActiveChannel prototype but not the Channel one until a later bug fix, but that bug fix didn't get backported all the way. That was part of the reason for that discussion about a ponyfill for diagnostics_channel rather than a polyfill.

Reply:
The check for subscribers has been added